spec/features/find/search/course_results_spec.rb

```diff
@@ -44,9 +44,9 @@ def i_see_the_courses
       # list by provider?
       expect(first_course.course_name.text).to include('Hello there')
       expect(first_course.provider_name.text).to be_present
-      expect(first_course.qualification.text).to include('QTS with PGCE')
-      expect(first_course.course_length.text).to eq('1 year - full time')
-      expect(first_course.funding_options.text).to eq('Teaching apprenticeship - with salary')
+      expect(first_course).to have_text('QTS with PGCE')
+      expect(first_course).to have_text('1 year - full time')
+      expect(first_course).to have_text('Teaching apprenticeship - with salary')
```

Comment:
If you wanted to use something like `first_course.qualification.text` they are defined here `spec/support/page_objects/find/results.rb` took me a while to find them in this [PR](https://github.com/DFE-Digital/publish-teacher-training/pull/4387/files#diff-b62fe3d6d4b01323426d7a73e90b2e11364fee2b4d9d45cc4a47e7f2296d3228) 🤦‍♂️